src/cgroups.cpp

```diff
@@ -30,14 +30,14 @@ void init_cgroup(uint64_t MEM_SIZE, double TOTAL_CPU_PERCENTAGE,
     switch (_cversion) {
     case 2: { // cgroup v2
         if (!FILE_EXISTS(MAIN_CGROUP_PATH))
-            SAFE("mkdir err:", mkdir(MAIN_CGROUP_PATH.c_str(), 0777));
+            SAFE("mkdir err:" + MAIN_CGROUP_PATH, mkdir(MAIN_CGROUP_PATH.c_str(), 0777));
```

Comment:
в одной строке пробел есть, в другой - нет
```suggestion
            SAFE("mkdir err: " + MAIN_CGROUP_PATH, mkdir(MAIN_CGROUP_PATH.c_str(), 0777));
```